packages/remediations/src/utils/utils.js

```diff
@@ -156,7 +156,7 @@ export const submitRemediation = (formValues, data, basePath, setState) => {
     api.patchRemediation(id, { add, auto_reboot: formValues[AUTO_REBOOT] }, basePath) ||
     api.createRemediation({ name: formValues[SELECT_PLAYBOOK], add, auto_reboot: formValues[AUTO_REBOOT] }, basePath)).then(({ id }) => {
         setState({ id, percent: 100 });
-        data.onRemediationCreated({
+        data.onRemediationCreated && data.onRemediationCreated({
```

Comment:
Optional chaining mught be better.
```suggestion
        data?.onRemediationCreated({
```

Reply:
@karelhala is it really applicable in this case with a function? 
![001](https://user-images.githubusercontent.com/50696716/122718146-dd895d00-d26c-11eb-97e1-a06e5c44303f.png)

Reply:
@karelhala that does not work on functions

Reply:
Oh, I forgot the other conditonal one as well
```suggestion
        data?.onRemediationCreated?.({
```

Reply:
@karelhala oh, great! Didn't know it works like this as well, thank you :-)